src/python/deepgnn/graph_engine/data/citation.py

```diff
@@ -264,48 +263,34 @@ def _write_json_graph(
         assert node_size == labels.shape[0]
         with open(graph_file, "w") as fout:
             for nid in range(node_size):
-                tmp = self.to_json_node(
+                tmp = self.to_edge_list_node(
                     nid,
                     node_types[nid],
                     flt_feat=list(features[nid].tolist()),
                     label=int(labels[nid]),
                     neighbors=adj[nid].nonzero()[1],
                 )
                 fout.write(tmp)
-                fout.write("\n")
 
     NODE_TYPE_ID = {"train": 0, "val": 1, "test": 2, "other": 3}
 
-    def to_json_node(
+    def to_edge_list_node(
```

Comment:
why this change is needed?

Reply:
It makes the data loaders about twice as fast.

Reply:
This is a public method, pease update the changelog.